.github/workflows/push.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
 
     - name: Docker meta
       id: docker_meta
-      uses: crazy-max/ghaction-docker-meta@v1
+      uses: docker/metadata-action@v3
```

Comment:
Is there a reason we need to change docker meta?

Reply:
Oops, just saw your comment, do we know for sure the tags will not change between the versions?

Reply:
right, it seems that docker took over the action as the official owner, but the code is the same. The v1 -> v2 upgrade notes are [here](https://github.com/docker/metadata-action/blob/v2.0.0/UPGRADE.md). AFAICT, the v1 -> v2 jump would not affect the behavior that's used in this repo. Then the [v2 -> v3 jump](https://github.com/docker/metadata-action/releases/tag/v3.0.0) was largely around the move to the new repo name.